daml-lf/integration-test-lib/src/main/com/daml/lf/CantonRunner.scala

```diff
@@ -76,6 +76,7 @@ object CantonRunner {
                |      admin-api.port = ${adminPort.port}
                |      ledger-api{
                |        port = ${ledgerApiPort.port}
+               |        explicit-disclosure-unsafe = ${config.enableDisclosedContracts}
```

Comment:
Maybe we just enable this always?

Reply:
Good question. Let's see  that latter though.